src/models/neural_networks/base.py

```diff
@@ -129,7 +129,15 @@ def train(self, num_epochs: int = 1,
                 val_rmse = []
                 with torch.no_grad():
                     for x, y in val_dataloader:
-                        val_pred_y = self.model(x[0], self._one_hot_months(x[1]))
+                        if self.experiment == 'nowcast':
+                            current = x[2]
+                            val_pred_y = self.model(
+                                x[0], self._one_hot_months(x[1]), current
```

Comment:
@tommylees112 , is this `if` statement necessary?

Sorry I didn't catch it before, but shouldn't the model not care if current is passed - it knows to ignore it if the experiment isn't `nowcast`.

It's similar to how the one hot months are passed to the model whether or not the model will use them, and the model decides whether or not to concatenate them to `x`.

We could reduce 10 lines of code from each model training function.

Reply:
Fixed in https://github.com/esowc/ml_drought/pull/66/commits/19d7b41e1b95d2dbe5c40e96be85bb6126d12e63